### DISCUSSION TITLE
How to apply color definitions for fmt.fg and fmt.bg

### DISCUSSION THREAD

**Discussion by khsoh** [kovidgoyal/kitty#7976] [Q&A]
In the theme configuration, colors like color0, color1, etc are defined.

How can I use these definitions in fmt.fg.XXX ?  I tried fmt.fg.color0 - it did not work.

**Comment by kovidgoyal** [comment]
fmt.fg.1 -- fmt.fg.9

**Comment by kovidgoyal** [comment]
Actually no that wont work because pythin doesnt like attribute names that start with numbers, this should take care of it: https://github.com/kovidgoyal/kitty/commit/d31459b0926f2afddc317d76314e4afd0d07d473
